docs/exceptions.md

```diff
@@ -137,83 +137,83 @@ treated formally or informally as **expected failures**.
 Let's recap. We've split all possible paths in programs to three groups:
 
 - **Expected success** is the main happy path of the program,
-  it represents how program behaves when everything goes right.
+  it represents how the program behaves when everything goes right.
 - **Expected failure** is secondary path that represent
-  all expected deviations from happy path e.g., when user gives an incorrect input.
+  all expected deviations from the happy path e.g., when a user gives an incorrect input.
 - **Unexpected failure** is some *unexpected* deviations from main or secondary paths,
   something that we call "bugs".
 
 
 ## try..catch
 
 How does `try..catch` fits into our three code paths groups view? It's great for unexpected failures!
-Or we should say: `throw` great for unexpected failures if we never actually `try..catch`.
-It's very good for debugging. Debugger will pause on the exact line that throws.
-Also if we don't use debugger we still get nice stack trace in console etc. It's sad that in many
-cases when program goes through unexpected path instead of exception we end up with `NaN`
-being propagated through program or something like that. In these cases it's much harder to track
-down where things went wrong, much nicer when it just throws.
+Or we should say: `throw` is great for unexpected failures if we never actually `try..catch`.
+It's very good for debugging. The debugger will pause on the exact line that throws.
+Also if we don't use a debugger we can still get a nice stack trace in the console, etc. It's sad that in many
+cases when a program goes through an unexpected path instead of exception we end up with `NaN`
+being propagated through the program or something like that. In these cases it's much harder to track
+down where things went wrong, it's much nicer when it just throws.
 
 On the other hand `try..catch` is bad for expected failures. There're many reasons why, but let's
 focus on just one: *it's bad for expected failures because it's already used for unexpected ones.*
 We must handle expected failures, so we would need to `try..catch` function that uses
 `throw` for expected failure. But if we do that we'll catch not only errors that represent
 expected failures, but also random errors that represent bugs. This is bad for two reasons:
 
-1. we ruin nice debugging experience (debugger will no longer pause etc);
+1. we ruin nice debugging experience (the debugger will no longer pause etc);
 2. in our code that is supposed to handle expected failures we would need to also
    handle unexpected failures (which is generally imposible as shown in the next section).
 
-If throw is used for expected failures in some API, we should wrap into `try..catch` as little code as
+If a `throw` is used for expected failures in some APIs, we should wrap `try..catch` into as little code as
```

Comment:
Should't "into" be before `try..catch` ? I was trying to say than we wrap should wrap "as little code as possible" inside `try..catch`